plugins/statistics.koplugin/main.lua

```diff
@@ -174,7 +174,9 @@ function ReaderStatistics:initData()
     self.data.series = book_properties.series
 
     self.data.pages = self.view.document:getPageCount()
-    self.data.md5 = self:partialMd5(self.document.file)
+    if not self.data.md5 then
```

Comment:
Should it also check that it's not an empty string?

Reply:
On initialize `self.data.md5` should be `nil` so we don't need to check that string is empty.